skeleton/gitops-template/components/http/base/initialize-namespace.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       serviceAccountName: pipeline
       containers:
       - name: initialize-namespace
-        image: quay.io/redhat-appstudio/appstudio-utils:5bd7d6cb0b17f9f2eab043a8ad16ba3d90551bc2@sha256:8c7fcf86af40c71aeb58e4279625c8308af5144e2f6b8e28b0ec7e795260e5f7
+        image: quay.io/redhat-ai-dev/utils:latest
```

Comment:
Maybe use the sha256 hash instead of latest?

Reply:
It's tagged as `latest` here https://quay.io/repository/redhat-ai-dev/utils?tab=tags, you want it retagged as the hash I assume?

Reply:
🤦 I just realized this was ai-lab-template and not ai-lab-app, so this is already in the GitOps template

(Side note: you can get the sha256 hash directly from Quay, without retagging)

Reply:
Ah okay I think I found on Quay where you can reference the tag via hash instead of the tag name, thanks for letting me know that!